client/src/app/pages/main/header/header.component.html

```diff
@@ -1,5 +1,9 @@
 <section id="header">
+  
   <header class="masthead">
+    <div class="gradient">
+      <video class="banner-video" src="../../../../assets/images/bg-main.webm" poster="../../../../assets/images/bg-main.webp" loop autoplay ></video>
```

Comment:
Is this properly pre-loading? As in, the website loads fully and this asynchronously loads and then replaces the "poster" when it does?

Reply:
I believe so, but I actually think this could be axed. If a poster tag isn't included I believe it uses the first frame of <video> as the poster instead, and since it's the same source material we could optimize it further by leaving it out

Reply:
I think it's fine to keep the poster. I read about preload some more and in our use case, with `autoplay`, we'll want it to load immediately anyways. So this should be fine!